package.json

```diff
@@ -25,8 +25,10 @@
     "jshint": "^2.7.0"
   },
   "dependencies": {
-    "debug": "^2.1.3",
-    "noble": "^1.1.0",
-    "eddystone-url-encoding": "^1.0.0"
+    "bluetooth-hci-socket": "^0.5.1",
+    "debug": "^3.1.0",
+    "eddystone-url-encoding": "^1.0.1",
+    "noble": "^1.9.1",
```

Comment:
This library should be using `@abandonware/noble` and `@abandonware/bluetooth-hci-socket`.

Reply:
This was built with the previous repo